Automattic/vip-go-mu-plugins
# Wrapped Connection_Pilot\Controls::connect_akismet() in conditional

## Description
Added `VIP_AKISMET_SKIP_LOAD` condition to Connection_Pilot\Controls::connect_akismet() calls

## Changelog Description

Setting the "VIP_AKISMET_SKIP_LOAD" constant to true will prevent calls made to connect Akismet during the Jetpack Connect procedures in Cron and CLI.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1. Check out PR.
1. Set `VIP_AKISMET_SKIP_LOAD` constant to true in vip-config.php: `define( 'VIP_AKISMET_SKIP_LOAD', true );`
1. Run the Jetpack connect cli command: `wp jetpack-start connect`
1. Find the Jetpack connect cron entry ID: `wp cron-control events list`
1. Run the Jetpack connect cron event:  `wp cron-control events run [event id]`
